### ISSUE TITLE
Create and run test to ensure data is captured correctly

### ISSUE THREAD

**Issue by fongcindy** [GSA/px-benefit-finder#1421]
### Description

Nick P noticed on 6/11 a bug in the funnel data caused some inaccurate data. Most of the event data is gone after March 27. 

Let's look into creating a E2E testing that can ensure that the data is being captured correctly and working.

**Comment by scottqueen-bixal** [comment]
removing myself since we are tracking in another ticket, recommend updating this to conform more closely to consistent data reporting to prevent uknown/late loss of data - cc @fongcindy

**Comment by fongcindy** [comment]
@nickpistone Yes, the DoD is the QA for the data collection (+e2e test) and a weekly report out of the data to stakeholders

**Comment by fongcindy** [comment]
Might be a duplicate ticket, confirm
